jsapp/js/components/processing/analysis/responseForms/selectOneResponseForm.component.tsx

```diff
@@ -16,6 +16,9 @@ interface SelectOneResponseFormProps {
   uuid: string;
 }
 
+/**
+ * Displays a common header and radio input with all available choices.
+ */
```

Comment:
If I delete a single choice/radio option, it still shows up, even after exiting and re-entering the Analysis tab:

![Before:](https://github.com/kobotoolbox/kpi/assets/7819986/0594fa4e-4b23-417e-babf-ad143b3795f3)
![After:](https://github.com/kobotoolbox/kpi/assets/7819986/33e1d387-9e67-4325-8e8f-7f34a3543e6e)

It looks like the Select Multiple/checkbox fields hide deleted options in the Analysis tab, though.

Reply:
I forgot to filter out choices with `options.deleted` flag for `qual_select_one`…